src/js/pm-image-editor.js

```
@@ -226,6 +226,25 @@
             };
 
             /**
+             * Update selection dimensions to new width and height.
+             * If width and height are correct -- reset all transformations to avoid positions issues.
+             *
+             * @param int width
+             * @param int height
+             */
+            ImageEditorFactory.prototype.updateSelectionDimensions = function(width, height) {
+                var parentSize = this.parentSize();
+                if (width <= parentSize.width && height <= parentSize.height) {
```

If the given dimensions are greater than the image, this is bad, so maybe throw an error?